cmd/internal/cli/inspect_linux.go

```diff
@@ -266,12 +276,12 @@ func getFileContent(abspath, name string, args []string) (string, error) {
 
 	cmd, err := exec.PipeCommand(starter, []string{procname}, Env, configData)
 	if err != nil {
-		sylog.Fatalf("%s", err)
+		sylog.Fatalf("%s: %s", err, cmd)
```

Comment:
I think this needs to be cmd.Args, not cmd itself.

Reply:
Thanks @jscook2345 I'll give this a go!